thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/MetricEntity.java

```diff
@@ -102,4 +104,15 @@ public static MetricEntity fromURN(String urn) {
   public static MetricEntity fromSlice(MetricSlice slice, double score) {
     return fromMetric(score, slice.getMetricId(), slice.getFilters());
   }
+
+  public static String buildMetricUrn(Map<String, Collection<String>> filterMaps, long metricId) {
```

Comment:
IMO, this function should not be in MetricEntity class because the input and return types are not MetricEntity.  Maybe put it in DetectionUtils?

Reply:
I think it makes sense to keep it here. I have updated the method to return a `MetricEntity` and renamed the method to `fromMetric` to be consistent with the other methods.